softvis3d-frontend/src/classes/status/StatusActionQueue.ts

```diff
@@ -9,6 +9,16 @@ export default class StatusActionQueue<T extends StatusAction> {
         this._queue.push(action);
     }
 
+    public copyAndRemove(action: T): StatusActionQueue<T> {
```

Comment:
I checked the implementation again and a new copy of the StatusActionQueue is only needed on status updates. These methods now return a new instance. What do you think?